.buildkite/pipeline.yml

```diff
@@ -197,6 +197,15 @@ steps:
     <<: *timeout
     command: "scripts/whitespace.sh --ci"
 
+  - label: ":book: check demo table"
+    command: "python scripts/demo_table.py --action=compare"
```

Comment:
I wonder if we should add (possibly in the script itself) output that makes it clearer what someone should do when CI breaks (ie instructing them how run the script locally, eg like we do for black)

Reply:
We do: https://buildkite.com/stellar/stellargraph-public/builds/2990

(The link in the PR description was pointing to the wrong place before)

Reply:
that looks perfect 👍 👍